src/views/wishlist.pug

```diff
@@ -6,6 +6,11 @@ mixin sharedInfoProp(key, label)
       th= lang(label)
       td= sharedInfo[key]
 
+//- For "add item" fields w/ table-based positioning, do not display borders between rows
+mixin addItemField
+  td(style="border-top: none; border-bottom: none;")&attributes(attributes)
+    block
```

Comment:
Could use feedback on this. I don't want to inadvertently style all of the other elements which are positioned by table - could maybe add a class/style for this but the scope is pretty limited.

Reply:
I may be a simple person, but I don't mind this form being on multiple rows. As such, this could just as a simple `.field`:

```
  .field
    label.label= lang('WISHLIST_PRICE')
    .control
      input.input(
        type='text',
        name='price',
        placeholder=lang('WISHLIST_PRICE')
      )
```

Reply:
Agree, I think this could just as well be vertical. Will change.

Reply:
Changed